spec/menu_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'menu'
+
+describe Menu do
+  let(:chinese_menu) { [{food: "Salt and Pepper King Prawn", price: 3.99, quantity_ordered: 0}, {food: "Pork Dumplings", price: 2.50, quantity_ordered: 0}]}
```

Comment:
Metrics/LineLength: Line is too long. [156/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)